docs/pages/upgrading/client-tools-autoupdate.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: Teleport Client Tools Automatic Updates
+description: Explains how to use Teleport client tools (`tsh` and `tctl`) auto-updates.
+---
+
+This documentation explains how to keep Teleport client tools like `tsh` and `tctl` up-to-date.
+Updates can be automatic or self-managed, ensuring tools are secure, free from bugs,
+and compatible with your Teleport cluster.
+
+## Why Keep Client Tools Updated?
```

Comment:
I would turn this heading into regular body text, since this is a very short section that can easily be part of the introductory paragraph.